packages/server/src/ui/app.css

```diff
@@ -97,6 +97,8 @@
 
   --page-sidebar-width: 260px;
   --page-sidebar-nav-item-height: 24px;
+  --page-sidebar-footer-font-size: 14px;
+  --page-sidebar-footer-margin: 25px;
```

Comment:
we try not to use bespoke spacing, we have `var(--base-spacing)` for margin and padding :)

Reply:
@patrickhulce I have accepted your CSS changes and I have removed --page-sidebar-footer-maring as we don't need it any more.